mmtrack/models/motion/kalman_filter.py

```diff
@@ -227,3 +227,60 @@ def track(self, tracks, bboxes):
         costs = np.stack(costs, 0)
         costs[costs > self.gating_threshold] = np.nan
         return tracks, costs
+
+
+@MOTION.register_module()
+class OCKalmanFilter(KalmanFilter):
```

Comment:
You already inherit it from `KalmanFilter`. Many codes have been duplicated. Do not need to re-write here. 

In the init function, you can direct `super().__init__(*args, **kwargs)`.

Reply:
Deleted this legacy class.